packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -210,20 +212,22 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = {
             },
             "& > svg": {
                 fill: lightModeNeutralForegroundRest,
-                [highContrastSelector]: {
-                    fill: "ButtonText",
-                },
+                ...highContrastForeground,
+            },
+            [highContrastSelector]: {
+                background: "none",
             },
         },
         "& $carousel_sequenceIndicator": {
             "&::before": {
                 background: lightModeNeutralFillStealthRest,
                 "border-color": lightModeNeutralOutlineRest,
+                ...highContrastButtonColorIndicator,
             },
             "&$carousel_sequenceIndicator__active": {
                 "&::before": {
                     background: lightModeNeutralFillStealthRest,
-                    ...highContrastHighlightBackground,
+                    ...highContrastHighlightColorIndicator,
                 },
             },
```

Comment:
I think this might be better as:

```typescript
"&::before: {
    "border-color": lightModeNeutralOutlineRest,
},
"&::before, &$carousel_sequenceIndicator__active": {
    background: lightModeNeutralFillStealthRest,
    ...highContrastButtonColorIndicator,
}
```